cm/example_multiply/README.md

```diff
@@ -16,10 +23,10 @@ Once EnerMaps launched, the frontend should be available on this adress : http:/
 
 ## Inputs 
 
-To make the CM work, we need thres inputs:
+To make the CM work, we need three inputs:
 * the raster we want to studied,
 * the areas we want to studied,
```

Comment:
"we want to study"

Reply:
I made the suggested changes

Reply:
I wanted to say:

To make the CM work, we need three inputs:
* the raster we want to study,
* the areas we want to study,

Reply:
sorry, I fixed the error in the last commit